### PR TITLE
ADEN-9211 Adjust Muthead ad unit and src

### PR DESCRIPTION
**Pull request by Bielik20**

### CODE REVIEW

File: src/ad-engine/models/targeting.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export interface Targeting {
```
Comment (Bielik20) [inline src/ad-engine/models/targeting.ts]: Please someone take a look at it. Maybe we don't need some of that stuff anymore.

File: platforms/mutfut/targeting.ts [hunk 2]
```diff
@@ -1,59 +1,33 @@
...
+	const cid = utils.queryString.get('cid');
+	const targeting: Partial<Targeting> = {
+		ae3: '1',
+		skin: 'turf',
+		uap: 'none',
+		uap_c: 'none',
+		s0: 'gaming',
+		s1: domain.name,
+		s2: 'main',
+		dmn: `${domain.name}${domain.tld}`,
```
Comment (nandy-andy) [inline platforms/mutfut/targeting.ts]: What `tld` stands for?

File: src/ad-engine/models/targeting.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	ae3: '1';
```
Comment (nandy-andy) [inline src/ad-engine/models/targeting.ts]: We can get rid of `ae3` key-val I think. It was needed in the transition phase when for Oasis and mobile-wiki we had also ae2 :)

File: src/ad-engine/models/targeting.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	hostpre: string;
+	lang: string;
+	wpage: string;
+	ref: string;
+	esrb: string;
+	geo: string;
+	likho: string[];
+	pv: string;
+	cid: string;
+	top: '1k';
```
Comment (nandy-andy) [inline src/ad-engine/models/targeting.ts]: The rest look fine. I'm not sure about the `s2`, `s0v` and `s0c` but if it's consistent with what we did in Gamepedia it's fine :)

File: platforms/mutfut/targeting.ts (hunk 2 above)
Comment (nandy-andy) [reply platforms/mutfut/targeting.ts]: 🙈 Got it! https://en.wikipedia.org/wiki/Top-level_domain 😊

**Pull request merged**